### PR TITLE
Add new events plugin - Synchronized Lyrics

### PR DESCRIPTION
**Pull request by elfalem**
This plugin reads .lrc files and shows timed lyrics.

### CODE REVIEW

File: quodlibet/quodlibet/ext/events/synchronizedlyrics.py [hunk 1]
```diff
@@ -0,0 +1,305 @@
...
+        s.set_numeric(True)
+        s.set_text(str(self.getFontSize()))
+        t.attach(s, 1, 2, 4, 5)
+        s.connect('value-changed', self.setFontSize)
+
+        vb.pack_start(t, False, False, 0)
+        return vb
+
+    def getTextColor(self):
+        v = config.get("plugins", self.PLUGIN_ID + "_txt",
```
Comment (declension) [inline quodlibet/quodlibet/ext/events/synchronizedlyrics.py]: There's actually a class to wrap all this (consistently). 
See [PluginConfigMixin](https://github.com/quodlibet/quodlibet/blob/master/quodlibet/quodlibet/plugins/__init__.py#L456).

File: quodlibet/quodlibet/ext/events/synchronizedlyrics.py [hunk 2]
```diff
@@ -0,0 +1,305 @@
...
+
+    def getBgColor(self):
+        v = config.get("plugins", self.PLUGIN_ID + "_bg", self._defaultBgColor)
+        return v[:3] + v[5:7] + v[9:11]
+
+    def getFontSize(self):
+        return int(config.get("plugins", self.PLUGIN_ID + "_fsize",
+                              self._defaultFontSize))
+
+    def setTextColor(self, button):
```
Comment (declension) [inline quodlibet/quodlibet/ext/events/synchronizedlyrics.py]: ..and for set etc.

File: quodlibet/quodlibet/ext/events/synchronizedlyrics.py [hunk 3]
```diff
@@ -0,0 +1,305 @@
...
+            #parse lyricsLine
+            if not lyricsLine[1].isdigit():
+                continue
+            closeBracket = lyricsLine.find("]")
+            timeObject = datetime.strptime(lyricsLine[1:closeBracket],
+                                           '%M:%S.%f')
+            timeStamp = (timeObject.minute * 60000 + timeObject.second * 1000
+                         + timeObject.microsecond / 1000)
+            words = lyricsLine[closeBracket + 1:]
+            if words == "":
```
Comment (declension) [inline quodlibet/quodlibet/ext/events/synchronizedlyrics.py]: Is `if not words`, or better, `if words` with reversed logic sufficient here? 

If not, depending on the difference between various _falsey_ things (empty string, `None`, `False` etc) is risky.

**Comment by declension** [comment]
Thanks. It looks good to me, other than a few tweaks. 

Can we ensure adherence to [PEP 8 naming](https://www.python.org/dev/peps/pep-0008/#method-names-and-instance-variables) - notably underscores for method names, not camel case.

**Comment by elfalem** [comment]
Thanks for the comments. I've followed PEP 8 conventions and used PluginConfigMixin.

I'm not sure I understand the comment about depending on falsey things. That area of the code will act on strings such as `[03:28.87][03:09.36][01:37.52][00:48.97]Some words` broken up by the brackets. The first three slices would match the empty string condition while the fourth one would fail.

**Comment by declension** [comment]
Great - thanks for the plugin.

**Pull request merged**